### ISSUE TITLE
overflow: undefined breaks OverlayScrollbarsComponent

### ISSUE THREAD

**Issue by krutoo** [KingSora/OverlayScrollbars#586]
**Describe the bug**
I use react version like this:
```jsx
<OverlayScrollbarsComponent
  defer={false}
  options={{
    overflow: undefined,
  }}
>
  {children}
</OverlayScrollbarsComponent>
```

TypeScript types say that overflow is not required options so there is no type check errors

But when i render it then it fails with error:

```
TypeError: Cannot read properties of undefined (reading 'x')
```

**To Reproduce**

1. Go to https://stackblitz.com/edit/overlayscrollbars-react-l1mfu8?file=src%2Findex.css
2. Click on Open in new tab
3. Open dev console
4. See errors "something is undefined"

**Expected behavior**
undefined value should be interpreted as not provided overflow option i think

**Comment by KingSora** [comment]
@krutoo thanks for the report :)

I'll take a look.. Im just curious, whats exactly the usecase here? What do you expext the `overflow` option to be when you specify it like this?

**Comment by krutoo** [comment]
@KingSora this is consistent with the work of javascript, for example, when destructuring objects, you can assign a default value that will be applied regardless of whether the object has an undefined value or is not defined at all:

```js
const source = { a: undefined };

const { a = 1, b = 2 } = source;

console.log(a, b) // 1 2
```

In my case i have component that passes a value to an internal component OverlayScrollbarsComponent:

```jsx
function MyComponent ({ overflow }) {
  return <OverlayScrollbarsComponent options={{ overflow }} />
}
```

Is optional prop, but MyComponent breaks when is not passed or passed as undefined

**Comment by KingSora** [comment]
I've released [`v2.4.4`](https://www.npmjs.com/package/overlayscrollbars/v/2.4.4) which should fix this issue.
